tests/gen_reactnative/test_combined/input.json

```diff
@@ -24,6 +24,56 @@
       }
     ]
   },
+  {
+    "type": "data",
+    "name": "DdSdkConfiguration",
```

Comment:
Let's not use this name in our test, it would bring some confusion.

Reply:
it was added to test the `export` exclusion in the generated files. but if we go for the `exposed` schema field, it is not really needed to have that name, yes.